packages/ibm-products/src/components/Coachmark/Coachmark.test.js

```diff
@@ -203,4 +203,28 @@ describe(componentName, () => {
       `${pkg.prefix}--coachmark__dark`
     );
   });
+
+  it('Check coachmark can be open by default', () => {
+    renderCoachmark({
+      'data-testid': dataTestId,
+      isOpenByDefault: true,
+    });
+    const coachmarkContainer = screen.getByTestId(dataTestId);
+    const coachmarkButton =
+      coachmarkContainer.children[0].children[0].children[0];
```

Comment:
Can you use something like `screen.getByLabelText('Show information')` instead? If the DOM structure of the coachmark changes at all, this test will fail.